### ISSUE TITLE
Automatically assign collaborators to new issues

### ISSUE THREAD

**Issue by johnnyman727** [tessel/project#28]
We need to automatically assign collaborators to issues on Tessel repos. Apparently the rust-lang group has open sourced the code they use to do that.

**Comment by Frijol** [comment]
@johnnyman727 do you still think this is a good idea? I don't.

**Comment by johnnyman727** [comment]
@Frijol I think it's a good way to make sure issues get labeled properly. If it's urgent the collaborator can hand it off to whomever can handle it best. If not, they can just label it and un-assign themselves. Why do you think it's not a good idea?

**Comment by Frijol** [comment]
If an issue is assigned to someone, I won't click on it and work on it, because I assume the assigned person is doing it. Better to have open issues that people can pick up and start working on.

It also fallaciously assumes that every issue is worth working on right away.

**Comment by johnnyman727** [comment]
Yes, the assigned collaborator should be responsible for assigning/un-assigning people and marking the urgency with a label. In any case, there are more urgent priorities than this at the moment :)

**Comment by Frijol** [comment]
Related to https://github.com/tessel/project/blob/master/meetings/2016-07-26.md#issue-triage cc @HipsterBrown @rwaldron @tcr @johnnyman727

**Comment by Frijol** [comment]
@tcr do you think this is closed as of us having @tessel-highfive ?
